--- origsrc/treecc-0.3.8/configure.in	2006-01-14 23:11:41.000000000 -0600
+++ src/treecc-0.3.8/configure.in	2006-08-10 12:17:43.468500000 -0500
@@ -8,10 +8,6 @@
 dnl Turn off the cygwin library if building for Win32.
 AC_MSG_CHECKING([if building for some Win32 platform])
 case "$host" in
-	*-*-mingw*|*-*-cygwin*)
-		platform_win32=yes
-		CC="gcc -mno-cygwin"
-		;;
 	*-*-mingw*)
 		platform_win32=yes
 		;;
